scripts/test_arduino_library.sh

```diff
@@ -60,8 +60,12 @@ InstallLibraryDependencies () {
 InstallLibraryDependencies
 
 for f in ${ARDUINO_LIBRARIES_DIR}/${LIBRARY_NAME}/examples/*/*.ino; do
-  echo "compiling $(basename ${f} .ino)"
+  EXAMPLE_NAME=$(basename ${f} .ino)
+  EXAMPLE_BINARIES=examples/${EXAMPLE_NAME}/binaries/${EXAMPLE_NAME}
```

Comment:
Actually, why do we need doubled EXAMPLE_NAME here? It seems a bit repetitive

Reply:
Also, maybe `output` instead of `binaries`. Please verify that we have a separate commit that adds this code, before any Renode stuff

Reply:
also please rebase after all is ready. We don't really need commits like "fix paths"

Reply:
Doubled `EXAMPLE_NAME` and `binaries` are taken from tensorflow-arduino-examples since it's how this path looks like there (see `--output-dir` flag): https://github.com/antmicro/tensorflow-arduino-examples/blob/master/build.sh#L23
If we don't want to follow that it can be easily changed

Reply:
yes, please reduce this

Reply:
Done